### DISCUSSION TITLE
Pimcore Academy - E-Commerce Framework

### DISCUSSION THREAD

**Discussion by vmalyk** [pimcore/pimcore#11035] [Q&A / Support] [answered]
Hi!
Can anyone clarify and check answers to Question1 about Principles of the E-commerce Framework.

<img width="483" alt="Снимок экрана 2021-12-16 в 09 43 26" src="https://user-images.githubusercontent.com/5318027/146321408-11b5a835-e7f4-4d15-8a8c-4b32a101337d.png">

I answered 2 points (based on documentation and video):

1.  Native integration into Pimcore
2. Provide a framework for developers to build e-commerce solutions for customers.

but it's marked as wrong-answered question.  Might be something wrong with question context.

**Comment by fashxp** [comment, marked as answer]
Hi, 
it also provides a set of modules that you can use in your implementation... 

BR
Christian
